### PR TITLE
unlink fence accounts

### PR DESCRIPTION
**Pull request by Shakespeared**
Allow permanent unlinking of fence accounts from profile or notification

### CODE REVIEW

File: src/pages/Profile.js [hunk 1]
```diff
@@ -117,24 +121,26 @@ const NihLink = ({ nihToken }) => {
...
-            div([h(ShibbolethLink, ['Log in to NIH to re-link your account'])])
-          ])
-        ]),
-        _.flow(
-          _.sortBy('name'),
-          _.map(renderDatasetAuthStatus)
-        )(datasetPermissions)
+    return div({ style: styles.identityLine }, [
+      h(Fragment, [
+        !linkedNihUsername && div({ style: styles.identityLine }, [ShibbolethLink, ['Log in to NIH to link your account']]),
```
Comment (panentheos) [inline src/pages/Profile.js]: It looks like something got lost in this edit. The `ShibbolethLink` won't render properly here.

File: src/pages/Profile.js [hunk 2]
```diff
@@ -117,24 +121,26 @@ const NihLink = ({ nihToken }) => {
...
-            div([Utils.makeCompleteDate(linkExpireTime * 1000)]),
-            div([h(ShibbolethLink, ['Log in to NIH to re-link your account'])])
-          ])
-        ]),
-        _.flow(
-          _.sortBy('name'),
-          _.map(renderDatasetAuthStatus)
-        )(datasetPermissions)
+    return div({ style: styles.identityLine }, [
+      h(Fragment, [
```
Comment (panentheos) [inline src/pages/Profile.js]: This `Fragment` isn't necessary, since there's a `div` that can contain the children.

File: src/pages/Profile.js [hunk 3]
```diff
@@ -207,17 +213,25 @@ const FenceLink = ({ provider: { key, name } }) => {
...
+            div({ style: { flex: 1 } }, ['Username:']),
+            div({ style: { flex: 2 } }, [username])
+          ]),
+          div({ style: styles.identityLine }, [
+            div({ style: { flex: 1 } }, ['Link Expiration:']),
+            div({ style: { flex: 2 } }, [Utils.makeCompleteDate(expireTime)])
+          ])
         ]),
-        h(FrameworkServiceLink, { linkText: 'Log-In to Framework Services to re-link your account', provider: key, redirectUrl })
+        div({ style: styles.identityLine }, [
```
Comment (panentheos) [inline src/pages/Profile.js:226]: If this `div` was inside the flex column just above, then we wouldn't need the additional `div` wrapper around everything. In general, we should strive for as few wrappers as possible to achieve the desired layout.

Comment (Shakespeared) [reply src/pages/Profile.js:226]: Removed a few extra divs here, which I think is what you mean. I can't see a way to remove the flex:1 and flex:2 divs without losing the formatting, but if there is a way (or if I messed up the formatting in a way I can't see) let me know!

File: src/pages/Profile.js (hunk 1 above)
Comment (Shakespeared) [reply src/pages/Profile.js]: good catch! Put this back the way it was.

**Review by panentheos — APPROVED** [review]
Looks good, tested locally.

**Pull request merged**